src/main/resources/application.properties

```diff
@@ -24,6 +24,10 @@ spring.data.mongodb.database=abdm_wrapper
 clientId=
 clientSecret=
 
+#HIP
+getPatientPath=http://localhost:8081/v1/patients
```

Comment:
nit: Can this be `hip.getPatientPath` to have more context?

Reply:
sure, grouping of properties should be in place. I have created a bug for this: https://github.com/NHA-ABDM/ABDM-wrapper/issues/36